### PR TITLE
Add command that configure China mirror

### PR DESCRIPTION
**Pull request by gucong3000**

### CODE REVIEW

**Comment by nschonni** [comment]
@gucong3000 our install process usually relys on GitHub releases for the binary downloads. Is that also something that is blocked? I know it can fall back to local build if necessary, I'm just curious of the experience with the alternate NPM install.

**Comment by xzyfer** [comment]
It'd be better to general document settings a proxy using npm config or Env
cars rather than suggesting a specific library and proxy server

**Comment by gucong3000** [comment]
> something that is blocked?

Yes, [Great Firewall](https://en.wikipedia.org/wiki/Great_Firewall) randomly dropp network packets from GitHub and Amazon S3 Service.

**Comment by xzyfer** [comment]
This seems reasonable to me. Curious to hear @EdwonLim thoughts since he maintains [node-sass-china](https://github.com/EdwonLim/node-sass-china).

File: README.md [hunk 1]
```diff
@@ -50,6 +50,13 @@ SyntaxError: Use of const in strict mode.
 
 **Having installation troubles? Check out our [Troubleshooting guide](/TROUBLESHOOTING.md).**
 
+### Install from mirror in China
+
+```shell
+npx mirror-config-china --registry=http://registry.npm.taobao.org
```
Comment (saper) [inline README.md]: Can this be done without adding `npx` dependency? Something like  

    npm i -g mirror-config-china --registry=https://registry.npm.taobao.org

Comment (gucong3000) [reply README.md]: Yes.

**Pull request merged**